src/app/myProfile/profileRead/ProfileRead.tsx

```diff
@@ -0,0 +1,51 @@
+import '../MyProfile.scss';
+import { Avatar, Button } from '@mui/material';
+import { deepPurple } from '@mui/material/colors';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+import { Profile } from '../profileUpdate/ProfileUpdate';
+
+interface ProfileReadProps {
+  editAction: () => void;
+  profile: Profile;
+}
+
+const ProfileRead = (props: ProfileReadProps) => {
+  return (
+    <div className='container'>
+      <Avatar
+        src='https://www.fit.pl/img/2007/02/_max/arnie.jpg'
```

Comment:
It should be mocked and passed as avatarUrl, u'll receive that in the future from backend dto